apparmor.d/profiles-a-f/briar-desktop

```diff
@@ -0,0 +1,97 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Roman Beslik <me@beroal.in.ua>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/briar-desktop
+profile briar-desktop @{exec_path} {
+  include <abstractions/audio-client>
+  include <abstractions/base>
+  include <abstractions/dconf-write>
+  include <abstractions/fontconfig-cache-read>
+  include <abstractions/gnome-strict>
+  include <abstractions/graphics>
+  include <abstractions/nameservice-strict>
+
+  network inet dgram,
+  network inet stream,
+  network inet6 dgram,
+  network inet6 stream,
+
+  @{lib}/jvm/java*/lib/jspawnhelper Cx -> jspawnhelper,
+
+  ptrace read peer=briar-desktop-tor,
+  ptrace read peer=@{profile_name}//jspawnhelper,
+
+  @{exec_path} mr,
+
+  @{lib}/jvm/java*/bin/java rix,
+  @{lib}/jvm/java*/lib/** rm,
+  @{sh_path} mr,
+
+  @{system_share_dirs}/icons/{,**} r,
```

Comment:
It  should be included in `gnome-strict`.

Reply:
All these lines?

Reply:
No only the one I selected (ie line 35, github shows too much)